aws-sns-topic/src/main/java/software/amazon/sns/topic/CreateHandler.java

```diff
@@ -24,10 +24,6 @@ protected ProgressEvent<ResourceModel, CallbackContext> handleRequest(
         final ResourceModel model = request.getDesiredResourceState();
         final Map<String, String> desiredResourceTags = request.getDesiredResourceTags();
 
-        if (StringUtils.isNotEmpty(model.getTopicArn())) {
```

Comment:
Remove this as stabilization will go through this again, while after creation topicArn has already been set in model.
This should only relies on schema check.